dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -162,20 +168,29 @@ public RelationshipItem getEnrollmentInRelationshipItem(
       return null;
     }
 
-    relationshipItem.setEnrollment(getEnrollment(enrollment, params, includeDeleted, currentUser));
+    relationshipItem.setEnrollment(
+        getEnrollment(
+            enrollment,
+            EnrollmentParams.FALSE.withIncludeAttributes(true),
```

Comment:
I am not sure we always want the attributes when getting an enrollment as part of a relationship 🤔 you should be able to come up with a `fields=relationships[from[...` that excludes them.

I think this should be `params.withRels(false)` so it respects what the user wanted but prevents us from getting relationships again 😅

Reply:
I agree.
But this is a wider fix, at the moment we are not passing the field parameters at all to the `getRelationshipItems` methods.
In this PR I just changed the hardcoded params to exclude the events that before were included.

Reply:
ok,as long as we do it before the release 😋